lib/backend/etcdbk/etcd_test.go

```diff
@@ -82,8 +83,6 @@ func (s *EtcdSuite) SetUpTest(c *check.C) {
 	c.Assert(err, check.IsNil)
 	s.bk = b.(*EtcdBackend)
 	s.suite.B = s.bk
-
-	s.bk.cfg.Key = legacyDefaultPrefix
```

Comment:
This causes a data race as the backend's `New` starts a process that also accesses the key